src/wp-includes/functions.php

```diff
@@ -9142,7 +9142,8 @@ function wp_fast_hash(
 	#[\SensitiveParameter]
 	string $message
 ): string {
-	return '$generic$' . sodium_bin2hex( sodium_crypto_generichash( $message ) );
+	$hashed = sodium_crypto_generichash( $message, 'wp_fast_hash_6.8+' );
+	return '$generic$' . sodium_bin2base64( $hashed, SODIUM_BASE64_VARIANT_URLSAFE_NO_PADDING );
```

Comment:
I think this is a good KISS approach for shortening the hash without losing data.

Is there any affect from introducing the key?

Reply:
The key isn't strictly necessary, but it provides domain separation. It doesn't affect the output length.